ts/components/settings/SessionNotificationGroupSettings.tsx

```diff
@@ -109,7 +109,7 @@ export const SessionNotificationGroupSettings = (props: { hasPassword: boolean |
             initialItem={initialNotificationEnabled}
             group={SettingsKey.settingsNotification}
             items={items}
-            onClick={async (selectedRadioValue: string | boolean) => {
+            onClick={async (selectedRadioValue: string) => {
               await window.setSettingValue(SettingsKey.settingsNotification, selectedRadioValue);
               forceUpdate();
             }}
```

Comment:
you can make the `initialNotificationEnabled` use your custom hook.
That way, you don't need the forceUpdate() for this case too.

Reply:
my bad, this is not be to be changed